MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationLayerController.java

```diff
@@ -304,7 +304,7 @@ public void onNewAnimationValue(Float value) {
       public void onNewAnimationValue(Float newPulseRadiusValue) {
         Float newPulseOpacityValue = null;
         if (options.pulseFadeEnabled()) {
-          newPulseOpacityValue = (float) 1 - ((newPulseRadiusValue / 100) * 3);
+          newPulseOpacityValue = 1.0f - newPulseRadiusValue / options.pulseMaxRadius();
```

Comment:
Can `options.pulseMaxRadius()` be 0?

Reply:
Good catch. `newPulseRadiusValue / options.pulseMaxRadius()` can never be more than 1.0f, it's in [0,1] here.